### ISSUE TITLE
Support for Matrox Monarch Edge 4 channel

### ISSUE THREAD

**Issue by qvestmedia** [bitfocus/companion-module-requests#111]
I would like to request the support for the Matrox Monarch Edge ( https://www.matrox.com/video/en/products/monarch_edge/specs/?ref=ad_BVE_blog_Monarch_Edge_article ) 

Use Case: Start and Stop the streaming of a given streamer output (GUID)

I can provide the API spec as personal mail.

**Comment by josephdadams** [comment]
Can you provide the API spec publicly here?

**Comment by qvestmedia** [comment]
Yes, it is. I am in the process of getting the approval from Matrox to paste the API Spec publicly here.

**Comment by WayneAndrewsMatrox** [comment]
Hi Joseph, 
Could you please email me in regards to the API you are requesting?
wandrews@matrox.com

**Comment by qm-admin** [comment]
Do you have a status on this? Matrox had provided the API spec to you some time ago.

**Comment by josephdadams** [comment]
They gave me the API but said I couldn’t post it or share it so it’s just sitting on my hard drive.

**Comment by McHauge** [comment]
but how can you implement it then, since it would become public when making the module?

**Comment by josephdadams** [comment]
They said they didn't mind if I made a Companion module; they just didn't want me sharing their API documents directly. Since it's not a device I use or plan to use, the effort to reward ratio isn't there for me. I don't have the time.
